peer/bind.go

```diff
@@ -119,3 +119,48 @@ func (c *BoundChooser) Introspect() introspection.ChooserStatus {
 	}
 	return introspection.ChooserStatus{}
 }
+
+// BindPeers returns a binder (suitable as an argument to peer.Bind) that
+// binds a peer list to a static list of peers for the duration of its
+// lifecycle.
+func BindPeers(ids []peer.Identifier) Binder {
+	return func(pl peer.List) transport.Lifecycle {
+		return &peersBinder{
+			once: intsync.Once(),
+			pl:   pl,
+			ids:  ids,
+		}
+	}
+}
+
+type peersBinder struct {
+	once intsync.LifecycleOnce
+	pl   peer.List
+	ids  []peer.Identifier
+}
+
+func (s *peersBinder) Start() error {
+	return s.once.Start(s.start)
+}
+
+func (s *peersBinder) start() error {
+	s.pl.Update(peer.ListUpdates{
```

Comment:
we should surface the error from this function, since Update does return an error.